lib/classes/tooltip.js

```diff
@@ -345,12 +313,40 @@ class ToolTip {
         this.$activeTrigger.focus = false;
         this.$activeTrigger.hover = false;
 
-        this.transition(tip, transEvent, complete, ClassName.FADE);
+        this.transition(tip, transEvent, complete);
 
         this.$hoverState = '';
     };
 
-    update() {
+    transition (tip, transEvent, complete) {
+        if (tip.classList.contains(ClassName.FADE)) {
```

Comment:
Why you remove 4rd arg, which allow to select ftansition class?

Reply:
The class is always FADE...  It is the only one that triggers transitions. SHOW doesn't

Reply:
Adding or removing SHOW triggers the transition, but fad stats that the tooltip has animation.

Reply:
K